argo-cd-bridges/gitlab/.helm/templates/cronjob.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "launcher.fullname" . }}
+spec:
+  schedule: "{{ .Values.cron.schedule }}"
+  successfulJobsHistoryLimit: {{ .Values.cron.historyLimit }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - image: docker-registry.default.svc:5000/{{ .Release.Namespace }}/{{ include "launcher.fullname" . }}:latest
```

Comment:
Can we reference the imagestreamtag instead of the image? IIRC the internal registry url changes in OCP v4.x.
Also I don't know if a `CronJob` can reference an imagestreamtag but there's a change that might be needed on the imagestream too (see comment below).